code/qcommon/vm.c

```diff
@@ -64,9 +64,9 @@ VM_Init
 ==============
 */
 void VM_Init(void) {
-    Cvar_Get("vm_cgame", "2", CVAR_ARCHIVE);	// !@# SHIP WITH SET TO 2
-    Cvar_Get("vm_game", "2", CVAR_ARCHIVE);	// !@# SHIP WITH SET TO 2
-    Cvar_Get("vm_ui", "2", CVAR_ARCHIVE);		// !@# SHIP WITH SET TO 2
+    Cvar_Get("vm_cgame", "0", CVAR_ARCHIVE);	// !@# SHIP WITH SET TO 0
```

Comment:
Perhaps make a comment here to indicate what the different values mean, or is this already commented somewhere where vm_ui etc is defined in one of those large structs? (if it is defined in a struct that is).